### ISSUE TITLE
Allow ic-prompt-value form key to be configurable

### ISSUE THREAD

**Issue by dotnetchris** [LeadDyno/intercooler-js#73]
The standard implementation definitely works with `ic-prompt-value` but it would be great if the value could be hoisted to my own specific name. Maybe something along the lines of:

```html
<button ic-put-to="/cust" ic-prompt="Id?" ic-prompt-parameter="customerId">Customer</button>
```

This would be very helpful for server side MVC frameworks' model binders.

Other suggestions off the top of my head would be along the lines of:

- ic-prompt-target
- ic-prompt-key 
- ic-prompt-name 
- ic-prompt-value (i don't really like this one due to overlap)

**Comment by carsongross** [comment]
fixed by f1468f8ce719e26807b5609ce92293f1a4689753

**Comment by dotnetchris** [comment]
For future readers, example:

```html
<button ic-put-to="/cust" ic-prompt="Id?" ic-prompt-name="customerId">Customer</button>
```
